### PR TITLE
Card section remove deprecated prop

### PR DESCRIPTION
**Pull request by Inbal-Tish**
## Description
CardSection - remove deprecated prop: 'sorce' (use 'imageSource').

## Changelog
CardSection - remove deprecated prop: 'sorce' (use 'imageSource').

### CODE REVIEW

File: src/components/card/CardSection.tsx [hunk 1]
```diff
@@ -36,7 +36,6 @@ export type CardSectionProps = ViewProps & {
    * Will be used for the background when provided
    */
   imageSource?: ImageSourcePropType;
-  source?: ImageSourcePropType; // TODO: remove after deprecation
```
Comment (ethanshar) [inline src/components/card/CardSection.tsx:39]: I found a usage in OneApp that still use the `source` prop. 
https://github.com/wix-private/restaurants/blob/af89e6db06c8e694e4d38719466ec88a93d5ffdc/one-app-restaurants-member/src/orders/screens/DispatchTypeScreen/components/DispatchTypeComponent.tsx#L51

Comment (Inbal-Tish) [reply src/components/card/CardSection.tsx:39]: That's wired. The change was already in the first V6 branch...

Comment (Inbal-Tish) [reply src/components/card/CardSection.tsx:39]: I asked @guywix to fix it.

Comment (ethanshar) [reply src/components/card/CardSection.tsx:39]: Did he respond? 
Does he have an ETA? 
We can still release v6 to community and Guy will have time till our next private version (which will take v6)

Comment (Inbal-Tish) [reply src/components/card/CardSection.tsx:39]: He didn't, but that's what I though as we didn't plan to release in private just yet

Comment (ethanshar) [reply src/components/card/CardSection.tsx:39]: Ok, but it's important to remember not to miss it. 
Anyway, ill approve

**Review by ethanshar — APPROVED** [review]

**Pull request merged**